AdvanceTaskMarsPart1/Tests/Notification_Tests.cs

```diff
@@ -0,0 +1,135 @@
+﻿using AdvanceTaskMarsPart1.Pages;
+using AdvanceTaskMarsPart1.Utilities;
+using AventStack.ExtentReports.Reporter;
+using AventStack.ExtentReports;
+using NUnit.Framework;
+using OpenQA.Selenium;
+
+namespace AdvanceTaskMarsPart1.Tests
+{
+    public class Notification_Tests : CommonDriver
+    {
+        LoginPage loginPageObj;
+        NotificationPage notificationPageObj;
+        public static ExtentReports extent;
+        public static ExtentTest test;
+
+        public Notification_Tests()
+        {
+            loginPageObj = new LoginPage();
+            notificationPageObj = new NotificationPage();
+        }
+
+        [OneTimeSetUp]
+        public static void ExtentStart()
```

Comment:
Also why have you put one time setup in all classes, it shows me you have not understood hooks please see this and fix https://dzone.com/articles/most-complete-nunit-unit-testing-framework-cheat-s-1